tests/unit/test_session_legacy.py

```diff
@@ -795,9 +795,7 @@ def test_internal_component(self):
             self.session._get_internal_component('internal'), component
         )
         with self.assertRaises(ValueError):
-            # get_component has been deprecated to the public
-            with pytest.warns(DeprecationWarning):
-                self.session.get_component('internal')
```

Comment:
For reference, we're handling the correct case [here](https://github.com/boto/botocore/pull/3109/files#diff-2d1a03ec5b80ddbfd9b9c273d7cace9e6c14b3b6facc646ee00c6d2587fc466bR815-R820).